samples/assistant/csharp-ooproc/AssistantApis.cs

```diff
@@ -56,33 +63,26 @@ public class CreateChatBotOutput
     /// HTTP POST function that sends user prompts to the assistant chat bot.
     /// </summary>
     [Function(nameof(PostUserQuery))]
-    public static async Task<PostResponseOutput> PostUserQuery(
+    public static async Task<HttpResponseData> PostUserQuery(
         [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "assistants/{assistantId}")] HttpRequestData req,
-        string assistantId)
+        string assistantId,
+        [AssistantPostInput("{assistantId}", "{message}", Model = "%CHAT_MODEL_DEPLOYMENT_NAME%")] AssistantState state)
```

Comment:
I recently learned from @fabiocav and @mattchenderson that binding to the message payload like this won't work in new .NET Isolated apps by default. Why may need to instead bind to query string parameters using the `{Query}` expression.

Reply:
updated to use {Query.message}